tooling/minimize/src/ty.rs

```diff
@@ -106,6 +111,13 @@ impl<'tcx> Ctxt<'tcx> {
                 let elem = GcCow::new(self.translate_ty(*ty, span));
                 Type::Slice { elem }
             }
+            rs::TyKind::Str => {
```

Comment:
Can you add some tests for `str`? Currently it seems only indirectly exercised.

Reply:
Can't, they get const promoted and unsized const pointers don't work currently.

Reply:
What is `str` even needed for then? panic_fmt is marked as a panic_fn so it should not need `str`.

Reply:
Idk maybe it isn't used by panic_fmt, but something else, but it doesn't build with `panic_fmt` marked, but no str support

Reply:
Then it should be possible to write a test.

Reply:
Ah, you do have tests now, never mind. :)